.github/workflows/create-release-branch.yml

```diff
@@ -7,27 +7,27 @@ on:
 jobs:
   create_branch:
     name: 'Create Branch'
-    
+
     runs-on: ubuntu-latest
 
     steps:
-    - name: Setup Node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - name: Checkout
-      uses: actions/checkout@v2.0.0
-      with:
-        ref: 'develop'
-    - name: Install Script Dependencies
-      run: npm install shelljs && npm install -g lerna
-    - name: Create and Push the Release Branch
-      run: |
-        git config --local user.email "releaseaction@github.com"
-        git config --local user.name "Release Action"
-        node scripts/create-release-branch.js
-      env:
-        SALESFORCEDX_VSCODE_VERSION: ${{ github.event.client_payload.version }}
+      - name: Setup Node.js
```

Comment:
Nit! Can we remove these extra spaces (I didn't see them in the previous version)? Everything looks good and I'll approve now.

Reply:
Yeah! It was forcing me to add them to save the file; let me clean that up.